x-pack/platform/plugins/shared/alerting/server/application/maintenance_window/methods/create/create_maintenance_window.ts

```diff
@@ -63,14 +63,6 @@ export async function createMaintenanceWindow(
     );
   }
 
-  if (scopedQueryWithGeneratedValue) {
```

Comment:
This validation was removed so that the new POST maintenance window API could be created.
The `categoryIDs` field will be completely removed in the future.

https://github.com/elastic/kibana/issues/197530